repo-docs/release-process.md

```diff
@@ -0,0 +1,113 @@
+# Release processes
+
+This repository contains two release processes, which are triggered on Git commits:
+
+## 1. The pre-release process
+
+The purpose of the pre-release process is to publish new changes from the main branch as frequently as possible. This has two main benefits:
+
+1. This tests whether we broke some parts of the release process so we don't get surprises when we need a manual release.
+2. This enables implementers to test a new feature or fix before we trigger a manual release.
+
+To achieve its purpose, the pre-release process is executed on every new commit on the main branch. Additionally, pre-releases do not commit anything to the main branch, which allows them to be deployed even if multiple features are merged on the main branch faster than the CI can release them.
+
+## 2. The manual release process
```

Comment:
Could we rename this to `The weekly release process` or `The scheduled release process` e.g. instead of `manual` to remove any confusion since it is automated now. And then we can refer to it as `the release` or `the scheduled release` in the text below.

Reply:
It's still almost entirely manual unfortunately. What's triggered weekly prompts us to approve it, and then often still requires intervention later on.

Still, I'm fine with `scheduled release process`, since we do it on a regular schedule, and I think that describes pretty well what users expect. I'll rename it.